cypress/component/ProgressReports/collaborator_changes.spec.tsx

```diff
@@ -44,10 +44,10 @@ describe('Collaborator Changes - Component Tests', () => {
 
   it('renders the component correctly', () => {
     cy.get('[data-cy=dar-closeout]').should('exist');
-    cy.contains('Step 2: Add or Remove Collaborators').should('be.visible');
-    cy.contains('2.1 Internal Lab Staff').should('be.visible');
-    cy.contains('2.2 Internal Collaborators').should('be.visible');
-    cy.contains('2.3 External Collaborators').should('be.visible');
+    cy.contains('Step 3: Add or Remove Collaborators').should('be.visible');
```

Comment:
To avoid making continual updates to these tests, they could use regular expressions. Assuming that the exact outline level is not something we need to test.
```suggestion
    cy.contains(/^Step \d+: Add or Remove Collaborators$/).should('be.visible');
```

Reply:
I agree that the tests are somewhat fragile due to the numbering scheme, but ultimately this form will change (and the numbers will probably be removed) which will require the tests to be updated anyways, so this seems like a reasonable stopgap until then.